samples/azure/azure-service-fabric-routing/Core_6/ServiceFabricRouting/ApplicationParameters/Local.5Node.xml

```diff
@@ -2,10 +2,10 @@
 <Application xmlns:xsd="http://www.w3.org/2001/XMLSchema" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="fabric:/ServiceFabricRouting" xmlns="http://schemas.microsoft.com/2011/01/fabric">
   <Parameters>
     <Parameter Name="ZipCodeVoteCount_PartitionCount" Value="1" />
-    <Parameter Name="ZipCodeVoteCount_MinReplicaSetSize" Value="3" />
-    <Parameter Name="ZipCodeVoteCount_TargetReplicaSetSize" Value="3" />
+    <Parameter Name="ZipCodeVoteCount_MinReplicaSetSize" Value="1" />
```

Comment:
This feels off... we should figure out why the sample is not working with the SF-based persistence, not scale down the partitions size.